sos/report/plugins/opencontrail.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2021 Mirntis, Inc., Oleksii Molchanov <omolchanov@mirantis.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenContrail(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """OpenContrail SDN
+    """
+    plugin_name = 'opencontrail'
```

Comment:
This needs a `short_desc` class attr which should have a short description of the plugin. Moving your docstring into `short_desc` is completely acceptable.

Reply:
Fixed.